compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2363,8 +2361,9 @@ object Types {
   }
 
   /** Assert current phase does not have erasure semantics */
-  private def assertUnerased()(implicit ctx: Context) =
+  private def assertUnerased()(implicit ctx: Context) = {
```

Comment:
Why did braces appear here ?

Reply:
One armed ifs are always enclosed in braces, in order to avoid accidentally merging them with an enclosing `else`.